.github/workflows/trivy_scan.yml

```diff
@@ -0,0 +1,71 @@
+name: Scan Antrea Docker image for vulnerabilities every day
+
+on:
+  schedule:
+    # every day at 10am
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.repository == 'antrea-io/antrea'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version-file: 'go.mod'
+    - name: Find greatest Antrea version
+      id: find-antrea-greatest-version
+      run: |
+        VERSION=$(git ls-remote --tags --ref https://github.com/antrea-io/antrea.git | \
+          grep -v rc | \
+          awk '{print $2}' | awk -F/ '{print $3}' | \
+          sort --version-sort -r | head -n 1)
+        echo "antrea_version=$VERSION" >> $GITHUB_OUTPUT
+    - name: Pull Antrea Docker images
+      id: pull
+      run: |
+        docker pull antrea/antrea-ubuntu:latest
+        docker pull antrea/antrea-ubuntu:${{ steps.find-antrea-greatest-version.outputs.antrea_version }}
+    - name: Run Trivy vulnerability scanner on latest Antrea Docker image
+      if: ${{ always() && steps.pull.conclusion == 'success' }}
```

Comment:
Out of curiosity, why these conditions are needed?
Job "pull" only runs when all previous jobs succeed. Then if Job "pull" runs and succeeds, this job will run anyway; if Job "pull" doesn't run or fails, this job won't run even without the conditions.
The comment applies to the other two jobs too.

Reply:
It's not needed for this step (`Run Trivy vulnerability scanner on latest Antrea Docker image`), but it is needed for the next step (`Run Trivy vulnerability scanner on Antrea Docker image for latest released version`) and the upload step. The idea is that even if this step fails because of detected CVEs, we still run the Trivy scan for the released image.

While it is not needed for this first step, I added it symmetry and that's the only reason.